### PR TITLE
[CircleGraph] Introduce 'scrollToSelected' message

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit introduces 'scrollToSelected' message.

Its correctness is tested in https://github.com/Samsung/ONE-vscode/pull/1543

Fresh draft: https://github.com/Samsung/ONE-vscode/pull/1543
Full draft: https://github.com/Samsung/ONE-vscode/pull/1505
Related: https://github.com/Samsung/ONE-vscode/issues/1491

ONE-vscode-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

**Review by dayo09 — APPROVED** [review]

File: src/CircleGraph/CircleGraphCtrl.ts [hunk 1]
```diff
@@ -55,6 +55,7 @@ export class MessageDefs {
   public static readonly backendColor = "backendColor";
   public static readonly error = "error";
   public static readonly colorTheme = "colorTheme";
+  public static readonly scrollToSelected = "scrollToSelected";
```
Comment (seanshpark) [inline src/CircleGraph/CircleGraphCtrl.ts:58]: I cannot accept this change and with related changes from the draft.
I need to test other modules with your change in the draft but cannot do the test as I am busy with other works and I cannot say when.

If possible, please do not fix this module but fix your codes.
If not, please post a new issue what you want, what the problem is with current codes.

Comment (stamalakhov) [reply src/CircleGraph/CircleGraphCtrl.ts:58]: @seanshpark 
I see. Thank you. ok.

Comment (seanshpark) [reply src/CircleGraph/CircleGraphCtrl.ts:58]: I've reviewed your draft code and there's no modification to existing features so I think it's OK to land the change.

**Review by seanshpark — APPROVED** [review]
LGTM

File: src/CircleGraph/CircleGraphCtrl.ts (hunk 1 above)
Comment (stamalakhov) [reply src/CircleGraph/CircleGraphCtrl.ts:58]: Ahh. ok. Thank you!

**Pull request merged**